DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: foundry
 Title: 'Palantir Foundry' Software Development Kit
-Version: 0.12.0
+Version: 0.13.0
```

Comment:
Should we autorelease here and just have CRAN track a subset of versions every now and then?

Reply:
What's the alternative?